### ISSUE TITLE
Show Recently Opened Folders in Windows Task-Bar Jump-List

### ISSUE THREAD

**Issue by stephenmartindale** [Microsoft/vscode#4698]
- VSCode Version: 0.10.11
- OS Version: Microsoft Windows 10

When you pin VS Code to the Windows Task-Bar and right-click on the task-bar icon, the jump-list shows only "New Window" and doesn't list recently opened folders - it should! And they should be pinnable!

![2016-03-27 vs code jump list](https://cloud.githubusercontent.com/assets/6370145/14066680/8a066588-f450-11e5-89ac-da7ee03660b0.png)

I really like the way that an opened folder replaces the full-fat Project concept of classic Visual Studio. By improving the Jump-List, things will be even better because working on multiple "projects" will become far more convenient.

**Comment by stephenmartindale** [comment]
After posting this feature request, I discovered [the UserVoice for the project](https://visualstudio.uservoice.com/forums/293070-visual-studio-code). (It's not linked in the [guidelines for contributing](https://github.com/Microsoft/vscode/blob/master/CONTRIBUTING.md) so I didn't know about it.)

I realised that feature requests shouldn't go here - sorry, all. I also realised that [jump-lists are supposedly already implemented](https://visualstudio.uservoice.com/forums/293070-visual-studio-code/suggestions/8844778-recent-jump-list-for-files-and-folder) and the UserVoice request for them is marked complete so what I am seeing is clearly a bug.
